client/src/components/DataFiles/DataFilesTable/DataFilesTable.jsx

```diff
@@ -94,7 +94,7 @@ const DataFilesTablePlaceholder = ({ section, data }) => {
         .
       </>
     );
-    if (err === '502') {
+    if (err === '500') {
```

Comment:
We should take this opportunity to disable key-pushing on systems that use the key service. I don't know if the situation has improved since Tapis V2, but prompting users to push keys to those systems could break things.

Reply:
I'd say let's do that in a follow on task. We do have a task for that already: https://jira.tacc.utexas.edu/browse/FP-1568